src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Entity
+@Table(name = "cinema_halls")
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
```

Comment:
in order to include the super class fields as well (id), you must add `@ToString(callSuper=true)`